### PR TITLE
Add cmd/request-bottle and delete old manual bottling scripts

### PR DESCRIPTION
**Pull request by issyl0**
- Supersedes #146 to use Homebrew.args instead of ARGV.

### CODE REVIEW

File: cmd/request-bottle.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      EOS
+    end
+  end
+
+  def request_bottle
+    request_bottle_args.parse
+
+    raise FormulaUnspecifiedError if Homebrew.args.named.empty?
+
+    formula = Homebrew.args.resolved_formulae
```
Comment (issyl0) [inline cmd/request-bottle.rb]: Do we need `.each` instead? Will we ever specify more than one formula to build with this command?

Comment (dawidd6) [reply cmd/request-bottle.rb]: `test-bot` itself seems to accept multiple formulae: https://github.com/Homebrew/homebrew-test-bot/blob/4b9881e72f7a02f4d8b8bfb594e472f7a01b4537/lib/test_bot.rb#L350-L369

Comment (dawidd6) [reply cmd/request-bottle.rb]: Or better not, one forumula per job seems enough and less error-prone.

Comment (dawidd6) [inline cmd/request-bottle.rb]:
```suggestion
    formula = Homebrew.args.resolved_formulae.last
```

Comment (issyl0) [reply cmd/request-bottle.rb]: `.first`?

Comment (dawidd6) [reply cmd/request-bottle.rb]: In previous PR it was `.last` :D

Comment (issyl0) [reply cmd/request-bottle.rb]: Fine!

**Pull request merged**